core/src/luigi-config/luigi-config.js

```diff
@@ -47,7 +47,7 @@ async function luigiAfterInit() {
     return;
   }
 
-  loadHiddenNamespacesToggle();
+  readFeatureToggles(['dontConfirmDelete', 'showHiddenNamespaces']);
```

Comment:
I think we should abstract away to `readAllFeatureToggles` - in this place we don't care about any particular toggle, we just want them all.

Reply:
the problem is we don't have a list of all toggles stored anywhere - so we need to provide a list of what to read from localStorage and assign to Luigi